Tensile/SolutionLibrary.py

```diff
@@ -478,17 +484,44 @@ def applyNaming(self, naming=None):
             s.name = OriginalSolution.getNameMin(s.originalSolution.getKernels()[0], naming)
 
     def remapSolutionIndicesStartingFrom(self, curIndex):
+        if self.lazyLibraries:
+            lazyLibrary = {}
+            for name, lib in self.lazyLibraries.items():
+                reIndexMap = {}
+                newSolutions = {}
+
+                for k, s in lib.solutions.items():
+                    reIndexMap[s.index] = curIndex
+                    s.index = curIndex
+                    newSolutions[curIndex] = s
+                    curIndex += 1
```

Comment:
Should add guard for chosen bucket size overflow, can use constant or 65536

Reply:
Bucket sizes are now 262144. Instead, I added a check for architecture clobbering.